doc/content/microreactors/index.md

```diff
@@ -5,3 +5,5 @@
 [SNAP 8 Experimental Reactor (S8ER)](s8er/index.md)
 
 [Gas-cooled Microreactor (GCMR)](gcmr/index.md)
+
+[Gas-cooled Microreactor (GCMR) Air Jacket](airjacket/index.md)
```

Comment:
let's nest airjacket under gcmr

Reply:
IMO I think it should be where it is currently. There's actually nothing about gas-cooled microreactors in this example; it's just CFD in a slot basically

Reply:
Maybe I'd replace this line by

```
[Air Jacket Passive Heat Removal](airjacket/index.md)
```